intelmq/bots/experts/certbund_contact/expert.py

```diff
@@ -71,13 +71,8 @@ def connect_to_database(self):
         self.logger.debug("Connected to PostgreSQL.")
 
     def process(self):
-        self.logger.debug("Calling receive_message.")
         event = self.receive_message()
 
-        if event is None:
```

Comment:
Is it _guaranteed_ that event can never be none here?  Removing handling of errors and exceptional situations like this is always dangerous...

Reply:
> intelmq.lib.bot.Bot.receive_message already logs in DEBUG and also
ensures the message is not None

https://github.com/certtools/intelmq/blob/0221a2507b05dcccaea9ac239bb9e13fa48ce4bc/intelmq/lib/bot.py#L650-L655